### PR TITLE
[vcpkg] allow --version to check the version

### PR DESCRIPTION
**Pull request by autoantwort**
I don't know if my approach is the best one.  
Fixes:
image
:D

### CODE REVIEW

**Comment by strega-nil** [comment]
requires:discussion on best way to actually do this

File: src/vcpkg/vcpkgcmdarguments.cpp [hunk 1]
```diff
@@ -260,6 +260,11 @@ namespace vcpkg
                 }
                 continue;
             }
+            if (basic_arg == "--version" && args.command.empty())
```
Comment (ras0219-msft) [inline src/vcpkg/vcpkgcmdarguments.cpp:263]: I'd suggest moving this up to the top as:
```cpp
if (arg_first != arg_last && arg_first + 1 == arg_last && *arg_first == "--version") {
    args.command = "version";
    return args;
}
```

Comment (autoantwort) [reply src/vcpkg/vcpkgcmdarguments.cpp]: Yeah that sounds better

**Comment by strega-nil** [comment]
Thanks @autoantwort

**Pull request merged**